### PR TITLE
[plugin.video.metv] 3.0.3

### PR DESCRIPTION
**Pull request by learningit**
### Description
V3.0.3 Fixes for website changes

### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.metv/lib/scraper.py [hunk 1]
```diff
@@ -13,124 +13,70 @@
...
          infoList = {}
-         infoList['MPAA']        = ''
-         infoList['TVShowTitle'] = name
-         infoList['Title']       = name
-         infoList['Studio']      = addonLanguage(30100)
-         infoList['Plot'] = h.unescape(plot.decode(UTF8))
-         meta['shows'][url] = (name, infoList)
-       ilist = self.addMenuItem(name,'GE', ilist, url, thumb, fanart, infoList, isFolder=True)
-     self.updateAddonMeta(meta)
+         infoList['TVShowTitle'] = showName
```
Comment (phil65) [inline plugin.video.metv/lib/scraper.py]: mediatype?

**Comment by phil65** [comment]
changelog.txt is shown as binary file?

**Comment by learningit** [comment]
Really? I'll have a look

**Comment by learningit** [comment]
@phil65 when I look at the file, even in github, it is clearly a text file. Any ideas why it's being classified as binary?

**Comment by phil65** [comment]
Correct encoding? Otherwise, no idea..

**Comment by learningit** [comment]
OK, fixed mediatype. I loaded, edited and saved the changelog.txt in notepad and it still shows as binary. I have no idea why, notepad is about as brain-dead as it gets and it reads fine. Squashed the commits as well.

**Comment by phil65** [comment]
Better try some text editor which can save with some explicit encoding. Notepad is always the worst choice to take.

**Comment by learningit** [comment]
I tried loading and saving with WordPad making sure it was set to ASCII and github didn't find any changes, so I assume it's the same. Really think it's some weirdness with github.

**Comment by phil65** [comment]
Wordpad is probaby the only one which is even worse :)
But sure, this isnt a showstopper anyways, so feel free to merge.

**Comment by learningit** [comment]
I only have a Windows laptop with.me so I can't be too snobby about editors.

**Pull request merged**